apps/condo/domains/news/constants/common.js

```diff
@@ -5,6 +5,9 @@ const conf = require('@open-condo/config')
 const SENDING_DELAY_SEC = Number(get(conf, 'NEWS_ITEMS_SENDING_DELAY_SEC', 15))
 const NEWS_SENDING_TTL_IN_SEC = Number(get(conf, 'NEWS_ITEM_SENDING_TTL_SEC', 60 * 30))
 const LOAD_RESIDENTS_CHUNK_SIZE = 20
+const MESSAGE_TITLE_MAX_LEN = 50
+const MESSAGE_BODY_MAX_LEN = 150
+
```

Comment:
Why do we need these consts?

Reply:
Because this constant use in task and test this task. One value in two different files